src/cloudWatchLogs/activation.ts

```diff
@@ -41,13 +41,6 @@ export async function activate(context: vscode.ExtensionContext, configuration:
             if (doc.isClosed && doc.uri.scheme === CLOUDWATCH_LOGS_SCHEME) {
                 registry.disposeRegistryData(doc.uri)
             }
-        }),
-        // Do highlight on text-changed event because we don't control when text is populated by vscode, also for "load more" codelens.
-        vscode.workspace.onDidChangeTextDocument((event: vscode.TextDocumentChangeEvent) => {
-            const eventUri = event.document.uri
-            if (registry.hasLog(eventUri) && !isLogStreamUri(eventUri)) {
-                registry.highlightDocument(eventUri)
```

Comment:
Looks like highlighting is being removed. Will this be added back?

Reply:
When I disabled this functionality I was still receiving highlights on the new text change when I did the 'Load More' codelens. I'm not sure if something changed with VSCode when this was initially implemented??

Reply:
I guess the logic was redundant.